rules/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromMockObjectRector.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\TypeDeclaration\Rector\ClassMethod;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr;
+use PhpParser\Node\Name\FullyQualified;
+use PhpParser\Node\Stmt\ClassMethod;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\ClassReflection;
+use PHPStan\Type\IntersectionType;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\Type;
+use PHPUnit\Framework\MockObject\MockObject;
```

Comment:
it seems got prefixed

```
use RectorPrefix202407\PHPUnit\Framework\MockObject\MockObject;
```
https://github.com/rectorphp/rector/commit/2ec758b3bcc3526b3795b178241f547b96ada4c9#diff-5e5ff8ef5c7ac56b67f99e00b5eed9d211c76942f1d55e8ac2854c5d164e3f12

Reply:
Could you fix these? I'm afk

Reply:
sure https://github.com/rectorphp/rector-src/pull/6167